Client/metrics.yaml

```diff
@@ -161,35 +170,10 @@ app_menu:
       - https://github.com/mozilla-mobile/firefox-ios/issues/12697
     data_reviews:
       - https://github.com/mozilla-mobile/firefox-ios/pull/12716
+      - https://github.com/mozilla-mobile/firefox-ios/pull/14102
     notification_emails:
       - fx-ios-data-stewards@mozilla.com
-    expires: "2023-06-01"
-  library:
-    type: counter
-    description: |
-        Counts the number of times a user taps Library in the app menu
-    bugs:
-      - https://github.com/mozilla-mobile/firefox-ios/issues/1544
-    data_reviews:
-      - https://github.com/mozilla-mobile/firefox-ios/pull/8167
-      - https://github.com/mozilla-mobile/firefox-ios/pull/9673
-      - https://github.com/mozilla-mobile/firefox-ios/pull/12334
-    notification_emails:
-      - fx-ios-data-stewards@mozilla.com
-    expires: "2023-06-01"
-  home:
-    type: counter
-    description: |
-        Counts the number of times a user taps Home in the app menu
-    bugs:
-      - https://github.com/mozilla-mobile/firefox-ios/issues/8147
-    data_reviews:
-      - https://github.com/mozilla-mobile/firefox-ios/pull/8167
-      - https://github.com/mozilla-mobile/firefox-ios/pull/9673
-      - https://github.com/mozilla-mobile/firefox-ios/pull/12334
-    notification_emails:
-      - fx-ios-data-stewards@mozilla.com
-    expires: "2023-06-01"
```

Comment:
AppMenu.library and AppMenu.home shouldn't be removed no? 🤔

Reply:
Unless I'm missing something, they don't exist anymore?
Or are we still using the same Home button probe even though it is in the toolbar now?

Reply:
Home is [still used](https://github.com/mozilla-mobile/firefox-ios/blob/3db84a24be39af6d453c02574addfcc45a6f4526/Client/Frontend/Browser/BrowserViewController/BrowserViewController%2BTabToolbarDelegate.swift#L17) in `tabToolbarDidPressHome` function of `BrowserViewController`. 

As for library it seems we can remove it, but we should clean up `TelemetryWrapper`
![Screenshot 2023-04-24 at 2 34 57 PM](https://user-images.githubusercontent.com/11338480/234085603-90a98516-d75f-4c74-ad17-aa22552a626e.png)